packages/components/pages/f-account-info/src/store/accountInfo.module.js

```diff
@@ -0,0 +1,48 @@
+import {
+    mapToConsumerDetails,
+    mapToConsumerAddress
+} from '../services/mapping/consumer.mapper';
+import {
+    UPDATE_CONSUMER_DETAILS,
+    UPDATE_CONSUMER_DETAIL
+} from '../constants';
+
+export default {
+    namespaced: true,
+
+    state: () => ({
+        consumer: {}
+    }),
+
+    actions: {
+        async loadConsumerDetails ({ commit }, { api, authToken }) {
+            const conversationId = api.setConversationId();
+            const getDetailsTask = api.getConsumerDetails(authToken, conversationId);
+            const getAddressTask = api.getConsumerAddresses(authToken, conversationId);
+
+            const responses = await Promise.all([getDetailsTask, getAddressTask]);
+
+            const details = mapToConsumerDetails(responses[0]?.data);
+            const address = mapToConsumerAddress(responses[1]?.data);
+
+            commit(UPDATE_CONSUMER_DETAILS, { details, address });
+        },
+
+        async editConsumerDetails ({ commit, state }, { field, value }) {
```

Comment:
is async needed here if nothing is being awaited?

Reply:
Good spot 👍